docker-compose.yml

```diff
@@ -53,6 +53,8 @@ services:
     container_name: backend-api
     build:
       context: backend
+      args:
+        BUILD_CONFIGURATION: Debug
```

Comment:
Build the docker container in **Debug** so that debugging can occur within the container.